FigureCollection.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace Lab_1
+{
+    class FigureCollection: List<Figure>

```

Comment:
Looks good. Why you need empty constructor for this?

Reply:
I deleted this file.